projects/evkmimxrt1060/test/.project

```diff
@@ -426,6 +426,16 @@
 			<type>1</type>
 			<locationURI>PARENT-3-PROJECT_LOC/examples/evkbmimxrt1060/test/semihost_hardfault.c</locationURI>
 		</link>
+		<link>
+			<name>source/test_logging.c</name>
+			<type>1</type>
+			<location>Y:/workspace/nxp-gri-idt/examples/evkbmimxrt1060/test/test_logging.c</location>

```

Comment:
Can we change this to relative paths?

Reply:
Opps I left one out. I will update the PR.